src/tempo.py

```diff
@@ -20,38 +20,48 @@ class Tempo:
     wal_path = "/etc/tempo_wal"
     log_path = "/var/log/tempo.log"
 
-    def __init__(
-        self, port: int = 3200, grpc_listen_port: int = 9096, local_host: str = "0.0.0.0"
-    ):
-        self.tempo_port = port
-
+    # todo make configurable?
+    receiver_ports: Dict[ReceiverProtocol, int] = {
+        "zipkin": 9411,
+        "kafka": 42,  # TODO:
```

Comment:
how about we make kafka/opencensus/thrift compact/binary a separate issue to avoid too much scope creep?